.gitignore

```
@@ -15,7 +15,7 @@ npm-debug.log*
 
 # Editor directories and files
 .idea
-.vscode
+.vscode/*.log
```

I don't know anything about the `.vscode` directory, but would it be better to do this?
```suggestion
.vscode/
!.vscode/settings.json
```